csrc/executor.cpp

```diff
@@ -1203,8 +1203,7 @@ std::vector<FusionExecutor::GlobalBufferInfo> FusionExecutor::
     GlobalBufferInfo info;
     info.tv = tv;
     info.zero_init = alloc->zeroInit();
-    std::tie(info.sizes, info.strides) =
-        inferShapeOfIntermediate(tv, alloc, expr_eval);
+    std::tie(info.sizes, info.strides) = inferShapeOfOutput(tv, expr_eval);
```

Comment:
`inferShapeOfIntermediate` is no longer used. I will remove it if we go this direction.